src/main/java/org/datavyu/util/Subject.java

```diff
@@ -0,0 +1,26 @@
+package org.datavyu.util;
```

Comment:
It's not very clear what this interface shall do from the doc-strings/description.

It also does not help that the return values are generic Objects.

Needs overhaul.

Reply:
My initial thought was to have only one getUpdate method and cast the result according to the event type (TIME_CHANGE, MIN_CHANGE, MAX_CHANGE) and we could expend that logic to any data type. But I think it is readable, easy and intuitive if we keep each message sent from an external clock separate.
I will update the description to reflect the usage of the interface.